code/game/machinery/portable_turret.dm

```diff
@@ -942,6 +934,20 @@ GLOBAL_LIST_EMPTY(turret_icons)
 		return
 	..()
 
+/obj/machinery/porta_turret_construct/screwdriver_act(mob/living/user, obj/item/I)
+	if(build_step != 6 && build_step != 5)
+		return
+
+	if(build_step == 5)
+		build_step = 6
+		to_chat(user, "<span class='notice'>You close the internal access hatch.</span>")
+	else
+		build_step = 5
+		to_chat(user, "<span class='notice'>You open the internal access hatch.</span>")
+
+	playsound(loc, I.usesound, 100, 1)
```

Comment:
I'm just gonna say that you should replace all of these direct playsound calls with `I.play_tool_sound(src)`